package.json

```diff
@@ -29,8 +29,56 @@
         "type": "git",
         "url": "https://github.com/shader-slang/slang-vscode-extension"
     },
-    "main": "./client/out/main",
+    "main": "./client/dist/nativeClientMain",
+    "browser": "./client/dist/browserClientMain",
     "contributes": {
+        "commands": [
+            {
+                "command": "slang.compile",
+                "title": "Slang Compile",
+                "category": "Slang",
+                "enablement": "editorLangId == slang",
+                "icon": "$(tools)"
+            },
+            {
+                "command": "slang.playgroundRun",
+                "title": "Run Playground",
+                "category": "Slang",
+                "enablement": "editorLangId == slang",
```

Comment:
Is it possible to associate a default hot key to this, e.g. F5?

Reply:
Would be nice if we can interpretable shader files and run `slangi` on the file if the file contains a `void main()` or `int main()`.

Reply:
I'm not familiar with `slangi`. Are docs for it somewhere?